docs/deployment/docker.md

```diff
@@ -169,11 +170,11 @@ RUN find packages \! -name "package.json" -mindepth 2 -maxdepth 2 -exec rm -rf {
 # Stage 2 - Install dependencies and build packages
 FROM node:16-bullseye-slim AS build
 
-# install sqlite3 dependencies
+# Install sqlite3 and isolated-vm dependencies, if you don't use sqlite you can remove libsqlite3-dev
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Comment:
sqlite3 and isolated-vm installation in the build stage in never reach. This whole RUN section can be removed with no issue

Reply:
I have high confidence that the `yarn install` at line 187/188 would then fail

Reply:
Going to test that now seeing as I have my test setup in place

Reply:
I get this error if I remove that entire section:  Error: Could not find any Python installation to use

Reply:
I see why : yarn install basically needs Python. I'll double check on my own Backstage repo history (as i'm deploying it without sqlite3) to see if I ever installated Python

Reply:
Pulling from the logs it's explicitly saying this: "error /app/node_modules/isolated-vm: Command failed". This all makes sense as Python3 is a requirement for isolated-vm. I did throughly test this to make sure it all worked as it's been a bit frustrating for the community and I wanted to be sure I could confidently refer people to this documentation. I'm often helping people on Discord so having correct docs makes that much easier.

Reply:
I confirm my doubts : before 1.15.0 and its ["BREAKING: Scaffolder build requirements"](https://backstage.io/docs/releases/v1.15.0#breaking-scaffolder-build-requirements), without sqlite3, Python was never required.

Reply:
Oh, also we don't use sqlite3 at all, not even on dev. It is not recommended for production use at all.

Reply:
Can I mark this as resolved?

Reply:
Close, just need to remove libsqlite3-dev as is mentionned [here ](https://github.com/backstage/backstage/pull/18612/commits/4f7292c74dff66790300c47f4b87888ab728511b#r1257390641)